### DISCUSSION TITLE
Specify location of done.txt, report.txt

### DISCUSSION THREAD

**Discussion by lamyergeier** [gsantner/markor#1691] [Ideas] [answered]
Todo.txt gives us flexibility to configure (in config file) separate location for todo.txt, done.txt and report.txt. This is useful, as I create a folder Todo and Done and keep these files separately in these folders as I have other documents relating to task in these folders as well.

Please let us choose the location of done.txt, report.txt as I would not want it to be the same as todo.txt

**Comment by harshad1** [comment, marked as answer]
@gsantner I think simplest way to enable this would be to allow relative paths.
